helpers.go

```diff
@@ -673,6 +673,14 @@ func (ctx *e2econtext) waitForMachinePoolUpdate(t *testing.T, name string) {
 	}
 }
 
+func (ctx *e2econtext) waitForStableCluster() error {
+	_, err := exec.Command("oc", "adm", "wait-for-stable-cluster", "--minimum-stable-period=2m").Output()
```

Comment:
The assumption here is that we don't care about the command output, just that it doesn't timeout waiting for a stable cluster.

Using a client library here in stead would be nice because it might give us more useful error messages without having to parse raw output.

Reply:
Per the testing, the remediation took about 25 minutes for a 6 node cluster.

Reply:
Could the cluster be modified to have a faster rollout? Machine config operate used to have such an option

Reply:
Yeah - that's a significant increase in our testing times. I'll do some digging around to see if there is a way to speed this up.